arch/arm64/configs/A6020_k5_defconfig

```diff
@@ -3671,7 +3671,6 @@ CONFIG_PANIC_ON_RECURSIVE_FAULT=y
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_PANIC_ON_RT_THROTTLING is not set
 CONFIG_SCHEDSTATS=y
-CONFIG_TIMER_STATS=y
```

Comment:
https://github.com/boulzordev/android_kernel_lenovo_msm8916/pull/12/commits/ab8916be1aa63dcd673ae6d7cc6308342239454d

no use to "is not set" when it gets removed fully :)